app/models/solr_document.rb

```diff
@@ -30,4 +30,9 @@ class SolrDocument
   # Recommendation: Use field names from Dublin Core
   use_extension( Blacklight::Solr::Document::DublinCore)    
 
+  def photo_path
```

Comment:
Cool! I didn't even think about that. It makes sense though. I'll change it and push it back up.